web/index.html

```diff
@@ -63,6 +63,7 @@
     <meta property="og:locale:alternate" content="ro_RO" />
     <meta property="og:locale:alternate" content="ru_RU" />
     <meta property="og:locale:alternate" content="sk_SK" />
+    <meta property="og:locale:alternate" content="sl_SI" />
```

Comment:
@madsnedergaard I assume we should and new languages here as well?

Reply:
Yup! And thanks for checking :)

Reply:
And in one more place, I'll just do it